File: back/src/main/java/csh/back/domain/vote/vote/dto/response/VoteCreateResponse.java
package csh.back.domain.vote.vote.dto.response;

import csh.back.domain.vote.vote.entity.Vote;

public record VoteCreateResponse(
        Long voteId
) {
    public static VoteCreateResponse from(Vote vote) {
        return new VoteCreateResponse(vote.getId());
    }
}

File: back/src/main/kotlin/csh/back/domain/vote/vote/dto/response/VoteCreateResponse.kt
package csh.back.domain.vote.vote.dto.response

import csh.back.domain.vote.vote.entity.Vote

data class VoteCreateResponse(
    val voteId: Long?,
) {
    companion object {
        @JvmStatic
        fun from(vote: Vote): VoteCreateResponse = VoteCreateResponse(vote.id)
    }
}
